.github/workflows/latest-bitcoind.yml

```diff
@@ -0,0 +1,51 @@
+name: Latest Bitcoin Core
+
+on:
+  schedule:
+    # Run at midnight on Sunday and Wednesday.
+    - cron: '0 0 * * 0,3'
+
+jobs:
+
+  regression-tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 90
+    steps:
+      - name: Checkout bitcoind master
+        uses: actions/checkout@v2
+        with:
+          repository: bitcoin/bitcoin
+          path: bitcoin
+
+      - name: Install bitcoind dependencies
+        run: sudo apt-get install build-essential libtool autotools-dev automake pkg-config bsdmainutils python3 libssl-dev libevent-dev libboost-system-dev libboost-filesystem-dev libboost-chrono-dev libboost-test-dev libboost-thread-dev libminiupnpc-dev libzmq3-dev libqt5gui5 libqt5core5a libqt5dbus5 qttools5-dev qttools5-dev-tools libprotobuf-dev protobuf-compiler git libsqlite3-dev ccache
+        working-directory: ./bitcoin
+
+      - name: Autogen bitcoind
+        run: ./autogen.sh
+        working-directory: ./bitcoin
+
+      - name: Configure bitcoind
+        run: ./configure
+        working-directory: ./bitcoin
+
+      - name: Build bitcoind
+        run: make -j "$(($(nproc)))"
```

Comment:
can do `"$(($(nproc)+1))"`

Reply:
Can you explain? That would give it our number of logical CPUs + 1, right? Why does it make sense to add one?

Reply:
tbh idk why but every compile bitcoin core from source tutorial I've seen uses +1

Reply:
I'd rather not include something I don't understand :wink: 
If you figure out why those tutorials do that, I'm interested!

Reply:
Seems it has to do with I/O delays and other resource constraints

https://unix.stackexchange.com/a/483306/330110

Reply:
Thanks for finding this!

> You may have some anecdotal experience where doing core-count + 1 results in faster compile times.

This doesn't look like a clear-cut win for doing +1 and feels highly dependent on the actual platform.
I'd rather stick to `nproc` which is intuitively the best choice, as +1 can in theory also degrade performance, and I don't want to have to measure the difference regularly to choose the best one only to win a few minutes of CI time (I would do it if it was running on every PR or commit to master, but this is going to be a nightly build that runs a few times a week).